content/en/docs/concepts/services-networking/service.md

```diff
@@ -534,6 +534,25 @@ to just expose one or more nodes' IPs directly.
 Note that this Service is visible as `<NodeIP>:spec.ports[*].nodePort`
 and `.spec.clusterIP:spec.ports[*].port`. (If the `--nodeport-addresses` flag in kube-proxy is set, <NodeIP> would be filtered NodeIP(s).)
 
+For example:
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: my-service
+spec:
+  type: NodePort
+  selector:
+    app: MyApp
+  ports:
+    - port: 80
+      # By default and for convenience, the targetPort is set to the same value as the port field.
+      targetPort: 80
+      # `targetPort` is the port of pod, you would like to expose
+      NodePort: 30007
+      # `NodePort` is the port of node, you would like to expose
```

Comment:
- should we move 553 before 552 to make sure comments appear before the stanza?
- should the `NodePort` be `nodePort`?
- is it possible that users may think that `nodePort` must be specified though it is usually auto selected?

Reply:
- true
 - It is my mistake
 - I think I should put more comments on it. thx

Reply:
updated, plz review it again. many thx

Reply:
Or do u think follow will be more clear?
```yaml
	apiVersion: v1
kind: Service
metadata:
  name: my-service
spec:
  type: NodePort
  selector:
    app: MyApp
  ports:
    - port: 80
      targetPort: 80
      # By default and for convenience, the `targetPort` is set to the same value as the `port` field.
      # nodePort: 30007
      # Optional field
      # By default and for convenience, the Kubernetes control plane will allocate a port from a range (default: 30000-32767)
```